students/shibin_mathew/lesson3/mailroom.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+import sys
+
+
+def optionOne():
+    # Function to Send a Thank You Email
+    quit = False
+    while not quit:
+        print("Please input your full name")
+        name = input()
+        if name == "list":
+            print("list of names")
+            for names in donors:
+                print(names['name'])
+            print("Please select from the list or add a new name")
+            name = input()
```

Comment:
`input` function accepts a string, this way instead of always having two lines this can be `input('Please select .. ')`

less unnecessary lines for reader to read improves readability of code.